x/programs/rust/wasmlanche-sdk/src/lib.rs

```diff
@@ -34,24 +35,6 @@ pub struct Context<K = ()> {
     pub action_id: Id,
 }
 
-impl<K> BorshSerialize for Context<K> {
-    fn serialize<W: std::io::prelude::Write>(&self, writer: &mut W) -> std::io::Result<()> {
-        let Self {
-            program,
-            actor,
-            height,
-            timestamp,
-            action_id,
-        } = self;
-        BorshSerialize::serialize(program, writer)?;
-        BorshSerialize::serialize(actor, writer)?;
-        BorshSerialize::serialize(height, writer)?;
-        BorshSerialize::serialize(timestamp, writer)?;
-        BorshSerialize::serialize(action_id, writer)?;
-        Ok(())
-    }
-}
```

Comment:
Why did you remove this?

Reply:
Because the `BorshSerialize` attribute does the same thing

Reply:
Can you put this back? This evaluates to 
```rust
impl<K: BorshSerialize> for Context<K> { // ...
```
But we don't need the trait bound on `K`.

Reply:
Sorry ! Indeed I expanded the code and it constraint `K` to be `BorshSerialize`, this is not the case on the automatic serialization for `Program` though. I'm backing this off.